code-challenges.test.js

```diff
@@ -10,53 +10,125 @@
+// // b) Create the function that makes the test pass.
 
+// it("takes in an array of numbers and returns an array with all the numbers multiplied by 3", () => {
+//     const numArr1 = [6, 7, 8, 9, 10]
+//     const numArr2 = [24, 27, 30, 33, 36]
+//      expect(multBy3(numArr1)).toEqual([18, 21, 24, 27, 30])
+//      expect(multBy3(numArr2)).toEqual([72, 81, 90, 99, 108])
+//    })
+ 
+// const multBy3 = (array) => {
+//    return array.map(value => value * 3)
+// } 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("numberDivisibleBy3", () => {
+    it("takes a number as an argument and decides if the number is evenly divisible by three or not.", () => {
+        expect(numberDivisibleBy3("is divisible by three")).toEqual("is not divisible by three")
+        const num1 = 15
+        const num2 = 0
+        const num3 = -7
+        })
+}) 
+// // Expected output: "15 is divisible by three"
+// // Expected output: "0 is divisible by three"
+// // Expected output: "-7 is not divisible by three"
+
+
+// // b) Create the function that makes the test pass.
+
+//PseudoCode
+    //create function to evaluate each numbers remainder after dividing by 3
+        // conditional statement, IF number % 3 = 0 then return true (divisible by 3)
+        // if not, return false  
+            //include speicific case for 0, cannot divide by 0
+
+const numberDivisibleBy3 = (number) => {
+    if (number === 0){
+      return "you cannot divide by zero"
+    } else if(number % 3 === 0 ){
+    return " is divisible by 3"
+     } else {
+        return "is not divisible by three"
+    }
+  }
 const num1 = 15
-// Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
-
-
-// b) Create the function that makes the test pass.
+console.log(numberDivisibleBy3(num1))
+console.log(numberDivisibleBy3(num2))
+console.log(numberDivisibleBy3(num3))
 
 
-
-// --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
+// // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("capitalString", () => {
+    it("takes in an array of words and returns an array with all the words capitalized.", () => {
+      const randomNouns1 = []
+      const randomNouns2 = []
+      expect(capitalString(randomNouns1))
+      expect(capitalString(randomNouns2))
+    })
+  })
```

Comment:
This is a great setup for a test, however, a test will always need a .toEqual method to compare the input to an expected output.  Currently, this test will pass everything that is passed in. Ex: `expect(capitalString(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])`